lib/components/normal-components/Board.ts

```diff
@@ -144,14 +144,4 @@ export class Board extends Group<typeof boardProps> {
   _computePcbGlobalTransformBeforeLayout(): Matrix {
     return identity()
   }
-
-  doInitialPcbDesignRuleChecks() {
-    if (this.root?.pcbDisabled) return
-    if (this.getInheritedProperty("routingDisabled")) return
-    const { db } = this.root!
-    const errors = checkEachPcbTraceNonOverlapping(db.toArray())
-    for (const error of errors) {
-      db.pcb_trace_error.insert(error)
-    }
```

Comment:
I think we want to only run this on the board, because DRC requirements are board-wide, it is a bit of an unusual case but i do think ots correct to have it on board